### PR TITLE
Events tweaking

### PR DESCRIPTION
**Pull request by Ziiro**
:cl: 
tweak: Potentially catastrophic event weightings reduced
tweak: Admins now have a 20 second warning on events to cancel them
balance: Greytide Virus can no longer target the engine
/:cl:

[why]: # Chaos Adjustment

### CODE REVIEW

File: code/modules/events/_event.dm [hunk 1]
```diff
@@ -60,8 +60,8 @@
 
 	triggering = TRUE
 	if (alertadmins)
-		message_admins("Random Event triggering in 10 seconds: [name] ([typepath]) (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
-		sleep(100)
+		message_admins("Random Event triggering in 20 seconds: [name] ([typepath]) (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
+		sleep(200)
```
Comment (FlattestGuitar) [inline code/modules/events/_event.dm]: replace that `200` with `20 SECONDS`

**Comment by AffectedArc07** [comment]
Can you make it so more events have sounds, like there are some events where the sounds dont play, either that or I lost my marbles.

File: code/modules/events/_event.dm (hunk 1 above)
Comment (ike709) [reply code/modules/events/_event.dm]: I like how you care more about using `SECONDS` than you care about the use of `sleep()`

File: code/modules/events/anomaly_grav.dm [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	name = "Anomaly: Gravitational"
 	typepath = /datum/round_event/anomaly/anomaly_grav
 	max_occurrences = 5
-	weight = 20
+	weight = 2
```
Comment (SamHPurp) [inline code/modules/events/anomaly_grav.dm]: wot

File: code/modules/events/electrical_storm.dm [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	typepath = /datum/round_event/electrical_storm
 	earliest_start = 6000
 	min_players = 5
-	weight = 30
+	weight = 10
```
Comment (SamHPurp) [inline code/modules/events/electrical_storm.dm]: This was literally already reduced in previous PR

File: code/modules/events/portal_storm.dm [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/portal_storm_syndicate
 	name = "Portal Storm: Syndicate Shocktroops"
 	typepath = /datum/round_event/portal_storm/syndicate_shocktroop
-	weight = 2
+	weight = 1
```
Comment (SamHPurp) [inline code/modules/events/portal_storm.dm]: I'd keep the same or even up a little bit but up the min players to 45 or something

File: code/modules/events/vent_clog.dm [hunk 5]
```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/vent_clog
 	name = "Clogged Vents"
 	typepath = /datum/round_event/vent_clog
-	weight = 35
+	weight = 15
```
Comment (SamHPurp) [inline code/modules/events/vent_clog.dm]: why reduce really low impact events? Kinda against the point

**Review by AndrewMontagne — APPROVED** [review]
LGTM

File: code/modules/events/anomaly.dm [hunk 6]
```diff
@@ -4,7 +4,7 @@
 
 	min_players = 1
 	max_occurrences = 0 //This one probably shouldn't occur! It'd work, but it wouldn't be very fun.
-	weight = 15
+	weight = 5
```
Comment (SamHPurp) [inline code/modules/events/anomaly.dm]: This does nothing

**Comment by SamHPurp** [comment]
So as things didn't move proportionally, guess what this has done?

Increased the frequency of which non-edited things occur. Including Blobs, Xenomorphs, Syndicate Shocktroopers...

**Pull request merged**